examples/composition/listing-component/resolvers.js

```diff
@@ -1,19 +1,33 @@
 'use strict';
 
+const GraphQLComponent = require('../../../lib');
+
 const resolvers = {
   Query: {
-    async listing(_, { id }, { dataSources }) {
-      const [property, reviews] = await Promise.all([
```

Comment:
Would be good to still have an example of utilizing the data source.

Reply:
technically this example does utilize a datasource - the top level component `listing-component` delegates down to reviews and property who make use of their own datasources